.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```

Comment:
You can just commit into repository Gradle Wrapper with executable permissions. The only reason for a wrapper to exist in a repository is to make the life of new developers in a team easier. It should be executable and should setup all the building things around in one click. Non-executable `gradlew` does not actually help anything...)

Reply:
`gradlew` was executable. It was just a useless step.